index.html

```diff
@@ -0,0 +1,637 @@
+          </div>
+          <div class="col-lg-4 us-main1 col-md-5">
+            <img src="img/2.png" alt="">
+            <a href="#">Conservative Author Warns Coronavirus 'Could Give Rise to the Antichrist'</a>
+          </div>
+
+          <div class="col-lg-4 us-aside d-flex flex-column col-md-2">
+            <div class=" us-side1 d-flex">
+              <a href="#"><img src="img/3.png" alt=""></a>
+              <p><a href="#">What Can You Do With Your Membership If the State Shuts Down Your Gym?</a></p>
+
+            </div>
+            <div class=" us-side1">
+              <a href="#"><img src="img/4.png" alt=""></a>
+              <p><a href="#">#YangWasRight Trends As Trump Eyes Giving Americans Checks Amid Pandemic</a></p>
+
+            </div>
+            <div class=" us-side1">
+              <a href="#"><img src="img/5.png" alt=""></a>
+              <p><a href="#">NY Stock Exchange Keeps Trading Floor Open Despite Coronavirus Pandemic</a></p>
+
+            </div>
+          </div>
+
+        </div>
+      </div>
+
+      <div class="container">
+        <div class="row">
+          <div class="us-heading">
+            <hr>
+            <h4>Sports</h4>
+          </div>
+          <div class=" col-lg-4 col-md-5 us-main ">
+            <img src="img/1.png" alt="">
+            <a href="#">Sanders Proposes $2,000 Monthly for Americans Until Coronavirus Threat Ends</a>
+          </div>
+          <div class="col-lg-4 us-main1 col-md-5">
+            <img src="img/2.png" alt="">
+            <a href="#">Conservative Author Warns Coronavirus 'Could Give Rise to the Antichrist'</a>
+          </div>
+
+          <div class="col-lg-4 us-aside d-flex flex-column col-md-2">
+            <div class=" us-side1 d-flex ">
+              <a href="#"><img src="img/3.png" alt=""></a>
+              <p><a href="#">What Can You Do With Your Membership If the State Shuts Down Your Gym?</a></p>
+
+            </div>
+            <div class=" us-side1 ">
+              <a href="#"><img src="img/4.png" alt=""></a>
+              <p><a href="#">#YangWasRight Trends As Trump Eyes Giving Americans Checks Amid Pandemic</a></p>
+
+            </div>
+            <div class=" us-side1 ">
+              <a href="#"><img src="img/5.png" alt=""></a>
+              <p><a href="#">NY Stock Exchange Keeps Trading Floor Open Despite Coronavirus Pandemic</a></p>
+
+            </div>
+          </div>
+
+        </div>
+      </div>
+    </section>
+
+    <section class="subscribe">
+      <h1>Hi</h1>
+      <div class="container">
+        <div class="row">
+          <hr>
+          <div class="col-lg-8 col-md-8">
+            <h4>SUBSCRIBE</h4>
+            <img src="img/12.png" alt="">
+
+          </div>
+          <div class="col-lg-4 col-md-4">
+            <h4>NEWSLETTER</h4>
+            <img src="img/11.png" alt="">
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] The elements in this section **should not be images** That defeats the whole purpose of the exercise and prevents you from learning how to position the elements when creating responsive designs. Please create the proper elements and adjust them accordingly.